handlers/grafana.go

```diff
@@ -69,29 +69,13 @@ func getGrafanaInfo(osRouteSupplier osRouteSupplier, serviceSupplier serviceSupp
 		VarWorkload:           grafanaConfig.VarWorkload,
 	}
 
-	// Case of overridden URL from config: use it and return immediately
-	if grafanaInfo.URL != "" {
-		return &grafanaInfo, http.StatusOK, nil
+	if grafanaInfo.URL == "" {
```

Comment:
Are these changes ok ?
I guess that we would like to check first if there is an URL defined via config, if not, then try to search it through the services API or openshift route API.
And I guess that Grafana and Jaeger services should use same mechanism.